Client/Client.CLI/App.cpp

```diff
@@ -9,80 +9,102 @@ App::~App()
+{
+    Base::Logger::FileLogger::getInstance().log("Server Accepted Connection", Base::Logger::LogLevel::INFO);
+}
 
-void App::onServerPing(double timestamp) { std::cout << "Ping: " << timestamp << "\n"; }
+void App::onServerPing(double timestamp) 
+{ 
+    Base::Logger::FileLogger::getInstance().log
+    (
+        "Ping: " + std::to_string(timestamp), 
+        Base::Logger::LogLevel::INFO
+    );
+}
 
-void App::onServerMessage(const uint64_t clientId) { std::cout << "Hello from [" << clientId << "]\n"; }
+void App::onServerMessage(const uint64_t clientId) 
+{ 
+    Base::Logger::FileLogger::getInstance().log
+    (
+        "Hello from [" + std::to_string(clientId) + "]", 
+        Base::Logger::LogLevel::INFO
+    );
+}
 
 void App::onChannelListRequest(const std::vector<Network::ChannelInfo>& channels)
 {
     for (const auto& item : channels)
     {
-        std::cout << item.channelName << '\n';
+        std::ostringstream out;
+        out << item.channelName;
+        Base::Logger::FileLogger::getInstance().log(out.str(), Base::Logger::LogLevel::INFO);
     }
 }
 
 void App::onMessageStoreAnswer(Utility::StoringMessageCodes storingMessageCode)
 {
     if (storingMessageCode == Utility::StoringMessageCodes::SUCCESS)
     {
-        std::cout << "SUCCESS sending" << std::endl;
+        Base::Logger::FileLogger::getInstance().log("SUCCESS sending", Base::Logger::LogLevel::INFO);
     }
     else if (storingMessageCode == Utility::StoringMessageCodes::FAILED)
     {
-        std::cout << "FAILED sending" << std::endl;
+        Base::Logger::FileLogger::getInstance().log("FAILED sending", Base::Logger::LogLevel::INFO);
     }
     else
     {
-        std::cout << "Unknown StoringMessageCode" << std::endl;
+        Base::Logger::FileLogger::getInstance().log("Unknown StoringMessageCode", Base::Logger::LogLevel::INFO);
     }
 }
 
 void App::onUserMessageDeleteAnswer(const Utility::DeletingMessageCodes deletingState) 
 {
     if(deletingState == Utility::DeletingMessageCodes::SUCCESS)
     {
-        std::cout << "SUCCESS deleting" << std::endl;
+        Base::Logger::FileLogger::getInstance().log("SUCCESS deleting", Base::Logger::LogLevel::INFO);
     }
     else if (deletingState == Utility::DeletingMessageCodes::FAILED)
     {
-        std::cout << "FAILED deleting" << std::endl;
+        Base::Logger::FileLogger::getInstance().log("FAILED deleting", Base::Logger::LogLevel::INFO);
     }
     else
     {
-        std::cout << "Unknown StoringMessageCode" << std::endl;
+        Base::Logger::FileLogger::getInstance().log("Unknown StoringMessageCode", Base::Logger::LogLevel::INFO);
     }
 }
 
 void App::onMessageHistoryAnswer(const std::vector<Network::MessageInfo>& messages)
 {
     for (const auto& item : messages)
     {
-        std::cout << item.message << '\n';
+        std::ostringstream out;
+        out << item.message;
+        Base::Logger::FileLogger::getInstance().log(out.str(), Base::Logger::LogLevel::INFO);
```

Comment:
Is it necessary to log each message? Maybe it is better to log only row such as "Message history recieved"